sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -226,6 +228,11 @@ def run(self, pipeline):
       raise ImportError(
           'Google Cloud Dataflow runner not available, '
           'please install apache_beam[gcp]')
+
+    # Add setup_options for all the BeamPlugin imports
+    setup_options = pipeline._options.view_as(SetupOptions)
+    setup_options.beam_plugins = BeamPlugin.get_all_plugin_paths()
```

Comment:
Did we want to augment the existing list (rather than overwrite it)? (If so, I suppose we should indicate it's comma delimited or similar.)